modules/javafx.fxml/src/main/docs/javafx/fxml/doc-files/introduction_to_fxml.html

```diff
@@ -720,7 +720,7 @@ <h4><a id="collections_and_property_handlers">Special handlers for collections a
 <p> Collections and object properties cannot be listen to using <span class="code">setOn<span class="variable">Event</span>()</span> methods.
     For these reason, special handler methods need to be used.
 <span class="code">ObservableList</span>, <span class="code">ObservableMap</span> or <span class="code">ObservableSet</span>
- uses a special <span class="code">onChange</span> attribute that points to a handler method with a <span class="code">ListChangeListner.Change</span>, <span class="code">MapChangeListener.Change</span> or <span class="code">SetChangeListener.Change</span> parameter respectively.
+ uses a special <span class="code">onChange</span> attribute that points to a handler method with a <span class="code">ListChangeListner</span>, <span class="code">MapChangeListener</span> or <span class="code">SetChangeListener</span> parameter, respectively.
```

Comment:
ListChangeListner --> ListChangeListener

Reply:
Also, the `.Change` was correct before your fix, unless I'm missing something.

Reply:
I misunderstood point 3 in the JBS issue. I thought that the change was to remove `.Change`.